tests/kernel/kirk.pm

```diff
@@ -30,11 +36,31 @@ sub run
     my $suite = get_var('KIRK_SUITE', '');
 
     select_serial_terminal;
+    zypper_call("ref");
+
+    my $cmd = '';
+    if ($install_from =~ /git/i) {
+        if (is_transactional) {
+            trup_install("git");
+        } else {
+            zypper_call("in -y git");
+        }
+
+        assert_script_run("git clone -q --single-branch -b $branch --depth 1 $repo");
+
+        $cmd .= "python3 kirk/kirk ";
+    } else {
+        add_ltp_repo;
```

Comment:
so we assume that kirk is in same repos as ltp, fair enough, but it should be mentioned is some doc section

Reply:
You mean doc section inside the code or somewhere else?